tamr_unify_client/models/project/estimated_pair_counts.py

```diff
@@ -0,0 +1,44 @@
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class EstimatedPairCounts(BaseResource):
+    """Estimated Pair Counts info for Mastering Project"""
+
+    @classmethod
+    def from_json(cls, client, resource_json, api_path=None) -> "EstimatedPairCounts":
+        return super().from_data(client, resource_json, api_path)
+
+    @property
+    def is_up_to_date(self) -> bool:
+        """Whether an estimate pairs job has been run since the last edit to the binning model.
+
+        :rtype: bool
+        """
+        return self._data.get("isUpToDate")
+
+    @property
+    def total_estimate(self) -> dict:
+        """The total number of estimated candidate pairs and generated pairs for the model across all clauses.
+
+        :return: Candidate pairs and estimated pairs with corresponding estimated counts
+        :rtype: dict[str, int]
+        """
+        return self._data.get("totalEstimate")
+
+    @property
+    def clause_estimates(self) -> dict:
+        """The estimated candidate pair count and generated pair count for each clause in the model.
+
+        :return: Each clause with corresponding estimated candidate pair count and generated pair count.
```

Comment:
Could you be more pedantic with the description of the format? "Each clause name mapped to a tuple containing the estimated pair count and generate pair count in that order." (I don't know what the actual format is off the top of my head, so I'm just making up the format for demonstration here).

An example of the format would be extremely helpful.